webui/react/src/routes/routes.ts

```diff
@@ -9,6 +9,24 @@ const routes: RouteConfig[] = [
     path: '/interactive/:taskId/:taskType/:taskName/:taskResourcePool/:taskUrl',
     title: 'Interactive Task',
   },
+  {
+    id: 'modelVersionDetails',
+    needAuth: true,
+    path: '/workspaces/:workspaceId/models/:modelId/versions/:versionNum/:tab',
+    title: 'Version Details',
+  },
+  {
+    id: 'modelVersionDetails',
+    needAuth: true,
+    path: '/workspaces/:workspaceId/models/:modelId/versions/:versionNum',
+    title: 'Version Details',
+  },
+  {
+    id: 'modelDetails',
+    needAuth: true,
+    path: '/workspaces/:workspaceId/models/:modelId',
+    title: 'Model Details',
+  },
```

Comment:
I split the URL for global model registry and workspace model registry to make things clear.

Reply:
We don't actually want separate urls, there should only be one canonical url for a model and model version.